modules/customize-certificates-replace-default-router.adoc

```diff
@@ -24,6 +24,8 @@ wildcard certificate must be the first certificate in the file. It can then be
 followed with any intermediate certificates, and the file should end with the
 root CA certificate.
 * Copy the root CA certificate into an additional PEM format file.
+* Verify all certificates that include `-----END CERTIFICATE-----` also
```

Comment:
```suggestion
* Verify that all certificates that include `-----END CERTIFICATE-----` also
```

Reply:
@mburke5678 Thank you for your review! I agree with your suggestion, but is there a way I can do it and not have the word "that" occur twice in one sentence?

Reply:
@mburke5678 I went for "Verify that all certificates which include ...."